lib/r10k/action/deploy/environment.rb

```diff
@@ -5,6 +5,7 @@
 require 'r10k/action/base'
 require 'r10k/action/deploy/deploy_helpers'
 require 'json'
+require 'tempfile'
```

Comment:
This is unused outside of tests now, right?

Reply:
Good catch.